configs/stage/roles/content-sources.json

```diff
@@ -0,0 +1,29 @@
+{
+  "roles": [
+    {
+      "name": "Content Sources administrator",
+      "description": "Perform any available operation against any Content Source resource.",
```

Comment:
since the app is branded 'Repositories', should we really put that in the name & description?